Frontend/src/pages/index.tsx

```diff
@@ -91,6 +92,26 @@ const MainPage = () => {
             // TODO
             return <></>;
         }
+
+        case PageType.TEXT_SELECTION: {
```

Comment:
nigdy prawda, przez cały flow użytkownika.

TextSelector powinen się pojawiać jak mamy otwarty quiz - w środku QuizPage

Reply:
Tak, tylko właśnie nie wiem w jaki sposób to tam dołączyć